sklearn/linear_model/_ridge.py

```diff
@@ -1969,7 +1969,8 @@ def fit(self, X, y, sample_weight=None):
 
         sample_weight : float or ndarray of shape (n_samples,), default=None
             Individual weights for each sample. If given a float, every sample
-            will have the same weight.
+            will have the same weight. The sample_weight vector should sum to
+            one.
```

Comment:
```suggestion
            will have the same weight. Note that the scale of `sample_weight`
            has an on the loss; i.e. multiplying all weights by `k`
            is equivalent to setting `alpha / k`.
```

Reply:
is word "impact" missed here? @glemaitre